docs/src/pubmed.md

```diff
@@ -122,20 +123,32 @@ db_path = "./test_db.db"
 
 const conn = SQLite.DB(db_path)
 PubMed.create_tables!(conn)
-PubMed.save_efetch!(conn, efetch_dict)
+PubMed.save_efetch!(conn, efetch_doc)
+```
+
+### Return efetch response as a dictionary of DataFrames
+
+The information returned by efetch can also be returned as dataframes. The dataframes match the
+format of the tables that are created for the sql saving functions (schema image below). These dataframes can also
+easily be saved to csv files.
+
+```julia
+    dfs = PubMed.pubmed_to_dfs(efetch_doc)
```

Comment:
It think it may be nice to keep the interface/naming consistent. What about calling the function `parse_efetch`?

Reply:
renamed all occurrences to PubMed.parse